dockstore-integration-testing/src/test/java/io/dockstore/webservice/UserResourceIT.java

```diff
@@ -513,4 +513,41 @@ public void testUpdateUserMetadataFromGithub() {
         assertEquals("DockstoreTestUser2", userProfile.getUsername());
     }
 
+    @Test
+    public void testSetUserPrivilege() {
+        ApiClient adminWebClient = getWebClient(ADMIN_USERNAME, testingPostgres);
+        ApiClient userWebClient = getWebClient(USER_2_USERNAME, testingPostgres);
+
+        PrivilegeRequest privilegeRequest = new PrivilegeRequest();
+        UsersApi adminApi = new UsersApi(adminWebClient);
+        UsersApi userApi = new UsersApi(userWebClient);
+        User admin = adminApi.getUser();
+        User user = userApi.getUser();
+
+        privilegeRequest.setAdmin(false);
+        adminApi.setUserPrivilege(user.getId(), privilegeRequest);
+        assertFalse(userApi.getUser().isIsAdmin());
+        assertFalse(userApi.getUser().isCurator());
+
+        privilegeRequest.setCurator(true);
+        adminApi.setUserPrivilege(user.getId(), privilegeRequest);
+        assertTrue(userApi.getUser().isCurator());
+
+        try {
+            userApi.setUserPrivilege(admin.getId(), privilegeRequest);
+        } catch (ApiException ex) {
+            assertEquals(ex.getCode(), HttpStatus.SC_FORBIDDEN);
```

Comment:
Think this will pass if no exception is thrown

Reply:
Not sure what you mean exactly. Im testing if the exception is thrown correctly if a curator tries to add admin privileges, since a curator should not be able to add/remove admin privileges.

Reply:
If your 
```
try {
  userApi.setUserPrivilege(admin.getId(), privilegeRequest);
}
```
was instead 
```
try {
  // Do absolutely nothing
}
```
your test still passes

Reply:
Ah gotcha, thanks for the clarification!